src/targets/gpu/fuse_ck.cpp

```diff
@@ -161,10 +203,81 @@ struct find_ck_gemm
     }
 };
 
+static bool is_mul_module(const module& m)
+{
+    std::vector<std::string> result;
+    for(auto& ins : m)
+    {
+        if(starts_with(ins.name(), "@"))
+            continue;
+        if(contains({"multibroadcast", "contiguous"}, ins.name()))
+            continue;
+        if(ins.name() == "pointwise")
+        {
+            return is_mul_module(*ins.module_inputs().front());
```

Comment:
Why is this called recursively?

Reply:
Meant to simplify that. I adapted it from another function

Reply:
Ok, I see, I dont think we have nested pointwise modules, so I dont think this will be needed.